modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -714,6 +714,74 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return 0.015;
   }
 
+  private getPsbtTxInputs(txHex: string) {
```

Comment:
(1) let's move this as a function to a separate file - if we don't plan to subclass a method, it could just be a function with argument `coin`

(2) let's make the signature take `UtxoPsbt` (or `string | UtxoPsbt` at least) to make it more versatile

Reply:
Done